EXAMPLES.md

```diff
@@ -126,9 +139,9 @@ In this example, if you called `product.title()` then an exception would be thro
 
 You can also make more complex requests using custom queries (see below).
 
-### Building a cart
+## Build a cart
```

Comment:
I think the "Cart line items based on selected options" can be a third-level heading while the checkout tasks should be separate from the Cart tasks.